server/html.c

```diff
@@ -4,12 +4,69 @@
 #include <sys/socket.h>
 
 #include <stdio.h>
+#include <stdlib.h>
 
 #include "../shared/constants.h"
 
+void parse_html_input(char* client_input, struct html_input* parsed_inputs, char mode) {
+    const char delimiter[] = "$";
+    char *token;
+
+    token = strtok(client_input, delimiter);
+
+    int no_parts;
+    if (mode == 'd') {
+        no_parts = 2;
+    } else if (mode == 'i' || mode == 'c') {
+        no_parts = 3;
+    } else {
+        // TODO: handle
```

Comment:
Vorschlag: Uf em server passiert nix und er sändet eifach a string vo wäge "Invalid usage of command" vo denne bi client aachunnt. Dä fangts ab und printets uf d Konsole

Reply:
ich han eher denkt, dass do e fehler catched werde muess wo mit developers mache. Entsprechend ka vo mir us gnaz gross uf de command line printed werde FAILURE und denn de programm executed werde

Reply:
terminated werde mein ich